### PR TITLE
implemented "Inserting multiple columns"

### PR DESCRIPTION
**Pull request by SaschaSchmidt**
some refactoring

Signed-off-by: Sascha Schmidt <sschmid@haw-landshut.de>

### CODE REVIEW

File: src/main/scala/com/campudus/tableaux/arguments.scala [hunk 1]
```diff
@@ -17,6 +17,8 @@ object ArgumentChecker {
 
   def notEmpty(x: Seq[_]): ArgumentCheck = if (!x.isEmpty) OkArg else FailArg("Argument is emtpy")
 
+  //def checkSeq(x: Seq[(Long, Any)]): Seq[ArgumentCheck] = x.foldLeft(Seq[ArgumentCheck]()) { (s, t) => s :+ greaterZero(t._1) } :+ x map { case (_, v) => notNull(v) }
```
Comment (SaschaSchmidt) [inline src/main/scala/com/campudus/tableaux/arguments.scala]: We have to check the content of the sequences. I hope u have a good idea

**Comment by Narigo** [comment]
Please add tests for bad input + try to generalize `checkSeq()`.
To use could be `checkArguments(checkSeq(someSeq): _*)`

**Comment by SaschaSchmidt** [comment]
fixed all @Narigo

**Comment by SaschaSchmidt** [comment]
Work done @Narigo

**Comment by Narigo** [comment]
We should add a different error message for casting exceptions, don't you think? It doesn't really make sense to say that there were not enough arguments when the real problem was that columnIds weren't numbers for example.

**Pull request merged**